docs/usage/configuration-options.md

```diff
@@ -1556,6 +1556,23 @@ To adjust it down to 10s for all queries, do this:
 }
 ```
 
+### httpsCertificateAuthority
+
+`httpsCertificateAuthority` overrides the trusted CA certificate that defaults to the curated list of well-known CAs by Mozilla.
+
+### httpsPrivateKey
+
+Specifies the private key in PEM format for mTLS authentication.
+
+<!-- prettier-ignore -->
+!!! warning
+    Do _not_ specify your private key directly here, to avoid losing confidentiality completely.
```

Comment:
```suggestion
    Do _not_ put your private key into this field, to avoid losing confidentiality completely.
```

How about an easier verb like `put` or `copy/paste` instead of `specify`?